INSTALL.md

```diff
@@ -1,11 +1,47 @@
+## Installation requirements on ubuntu
+
 WCIVF requires Python 3.
 
 To install:
 
     sudo apt-get install python3-dev libpq-dev libjpeg-dev redis-server
     pip install -r requirements/local.txt
 
-Create a Postgres database as detailed below, then:
+
+## Installation requirements on macOS
+
+Use homebrew to install the following dependencies:
+
+    brew install postgresql libjpeg-dev postgis
+
+Pyenv is recommended to install python versions. You can find installation instructions for pyenv [here](https://github.com/pyenv/pyenv#installation) - installation via homebrew is recommended. You can then install python 3 with:
+
+    pyenv install 3.6.12
+
+If using pyenv, after installation use the command `pyenv local 3.6.12` to set this version for the project locally. This will create a `.python-version` file with the specified python version.
+
+You can then create a [virtual environment](https://docs.python.org/3/tutorial/venv.html) with the command:
+
+    python -m venv env
+
+This will create a virtual environment in the directory `env`. You can then activate the virtual environment with:
+
+    source env/bin/activate
```

Comment:
This might not be helpful here, but you don't have to make your virtualenvs in this directory. I have a `~/.envs/` directory that I use for everything (e.g, `~/.envs/wcivf/`). This means I can e.g `grep` the project root without getting results from 3rd party packages (although this might be useful sometimes).

To manage the envs, I use `virtualenvwrapper` - just some shell scripts around the base `virtualenv` (there's also [`venvwrapper`](https://github.com/antebandov/venvwrapper)) - this allows me to activate the virtualenv by typing e.g `workon wcivf`.

Reply:
I'm gonna explore these myself and then think it could be a good thing to include in the tech handbook